pyramid_jinja2/scaffolds/jinja2_starter/+package+/templates/mytemplate.jinja2_tmpl

```diff
@@ -38,15 +38,15 @@
                 <span class="smaller">Jinja2 scaffold</span>
               </h1>
               <p class="lead">
-                {% trans %}Hello{% endtrans %} to <span class="font-normal">\{\{ project \}\}</span>, an&nbsp;application generated&nbsp;by<br>the <span class="font-normal">Pyramid Web Framework 1.5.4</span>.</p>
+                {% trans %}Hello{% endtrans %} to <span class="font-normal">\{\{ project \}\}</span>, an&nbsp;application generated&nbsp;by<br>the <span class="font-normal">Pyramid Web Framework {{pyramid_version}}</span>.</p>
```

Comment:
Both {{pyramid_version}} and {{pyramid_docs_branch}} are variables in the scaffold framework since https://github.com/Pylons/pyramid/commit/0eff6638d2470f77bd444941cb8261d91c236591

Escaping is not needed because pcreate renders the values, not the jinja2 template.

I've tested locally, and it works.